### DISCUSSION TITLE
UTM SE : Guest has not initialised the display (yet)ist of virtual boxses on ipad

### DISCUSSION THREAD

**Discussion by hussain626** [utmapp/UTM#6608] [UTM Help (iOS)]
I downloaded Ubuntu Version 14.04 and inserted into UTM SE. When I started the system it showed black screen and said “ Guest has not initialised the display (yet) “

**Configuration (required)**
* UTM Version:    UTM SE
* OS Version:      Ubuntu 14.04
* Device Model:   Ipad M1 pro
* Is it jailbroken:   No 
* How did you install UTM?    From App store 

I also tried using parrot OS security edition

**Comment by osy** [comment]
UTM SE is really slow and will probably take ~20 mins before anything shows up

**Reply by Intensity** [reply]
@osy : Could you provide further clarity here?

As in, do you happen to know if all VMs take this long to show anything at all?

That's at least two orders of magnitude of slowdown if so.

Could there be an alternative explanation for a VM to not show any display output? As in, improper Display settings, a silent qemu error that's not being verbosely displayed, an image that's unnecessarily switching graphics modes, using or not using UEFI boot when only the opposite works.

Does adding a serial device help?

Is there failsafe image or set of settings known to work? Some legacy systems stay in BIOS text mode so that may avoid unimplemented graphical modes, though the ultimate causes might be elsewhere.

**Reply by osy** [reply]
You should probably wait for v4.6.0 as there's been a lot of backend issues that's been fixed which may help in your case.
